.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Motivation
```

Comment:
I don't think this template applies for this repo.

What we're interested in when someone pushes a new RBI is the following:

* What is the gem being pushed?
* Where can we find the source code of this gem?
* Where can we find the api documentation of this gem?

If it's a change to an existing gem:

* What was the problem before the change?
* Can the problem be shown with an example or a sorbet.run?
* If you're adding/removing a definition, what is the change in the gem upstream that requires a change here?

etc.

Furthermore, there is not real "implementation", it's just a RBI file and the tests are automated, nothing to push here either.